webui/react/src/utils/types.ts

```diff
@@ -29,11 +29,14 @@ export const commandToTask = (command: Command): RecentTask => {
       date: command.registeredTime,
       name: 'requested',
     },
+    misc: command.misc,
     ownerId: command.owner.id,
+    startTime: command.registeredTime,
```

Comment:
question: `endTime`?

Reply:
just saw your comment about not having `endTime` for commands. What if we used `Date.now()` minus `startTime` as a start for duration in the meantime?

Reply:
hmm, duration wouldn't be part of the task type and might be tough to compute for command especially after they are no longer "running". Say a command is terminated we don't know when that happened => what would we show for duration?

Reply:
yep got it, let's nix it

Reply:
yep, let's nix duration.